### ISSUE TITLE
No data update

### ISSUE THREAD

**Issue by Boia11** [eelcohn/home-assistant-ddwrt#2]
In Home Assistant 0.115.3:
```
2020-09-25 22:07:20 WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for ddwrt doing I/O at custom_components/ddwrt/pyddwrt.py, line 448: response = self._session.get(
2020-09-25 22:07:20 WARNING (MainThread) [custom_components.ddwrt] Unable to update WAN data: list index out of range
2020-09-25 22:07:21 WARNING (MainThread) [custom_components.ddwrt] Unable to update wireless data: list index out of range
```

Configuration:
```
ddwrt:
    devices:
      - name: TP-LINK_Archer_C7
        host: 192.168.1.1
        username: !secret ddwrt_username
        password: !secret ddwrt_password
        ssl: true
        verify_ssl: false
```

**Comment by eelcohn** [comment]
I'm working on a new version which should be fully compatible with modern versions of HomeAssistant

**Comment by rlangtind** [comment]
I presume this is still an issue?  I also get this error on a  NightHawk R7800.

**Comment by eelcohn** [comment]
I’ve just published v1.1.0 of the DD-WRT integration on https://www.github.com/eelcohn/home-assistant-ddwrt. For anyone who has issues with v1.0.0, upgrade by doing:

    Create a backup of your configuration, including the custom_components folder
    Remove all DD-WRT integrations from Home Assistant
    Copy/install v1.1.0 of the DD-WRT integration
    Setup/configure the new component, either from the UI Integrations page, or from the configuration.yaml file

Please let me know if the update to v1.1.0 solves your problem.
